spot-contracts/contracts/BondIssuer.sol

```diff
@@ -102,7 +105,10 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
         for (uint8 i = 0; i < trancheRatios_.length; i++) {
             ratioSum += trancheRatios_[i];
         }
-        require(ratioSum == TRANCHE_RATIO_GRANULARITY, "BondIssuer: Invalid tranche ratios");
+
+        if (ratioSum > TRANCHE_RATIO_GRANULARITY) {
+            revert UnacceptableTrancheRatios();
```

Comment:
What do you think about parameterizing errors?
i.e.--
`revert SomeErrorName(expectedValue, actualValue);` or
`revert SomeInvalidValueErrorName(value)`

I'll guess that our basic structures involve a revert guarded by a conditional that tests theses values already, so they should be available in most cases.

Reply:
I don't really see any use to parameterizing errors right now. I could be useful, if we wanted to raise and catch exceptions downstream and do some more advanced handling; but don't see it being the case anytime soon.

Reply:
Sure. If there's value, it's in debugging runtime errors most likely.